Core/UINavigationController+STPTransitions.h

```diff
@@ -6,6 +6,10 @@
 
 - (void)pushViewController:(UIViewController *)viewController
            usingTransition:(STPTransition *)transition;
+
+- (void)popToViewController:(UIViewController*)viewController
+           usingTransition:(STPTransition *)transition;
```

Comment:
Haha, I don't mean to be _that guy_, but they're supposed to be underneath each other. Xcode does this automatically when you press the tab key.